meilisearch/_httprequests.py

```diff
@@ -19,7 +20,7 @@ def __init__(self, config: Config) -> None:
         self.config = config
         self.headers = {
             "Authorization": f"Bearer {self.config.api_key}",
-            "User-Agent": qualified_version(),
+            "User-Agent": self._build_user_agent(config.client_agents),
```

Comment:
```suggestion
            "User-Agent": _build_user_agent(config.client_agents),
````

Reply:
No because `_build_user_agent` shouldn't be part of the class

Reply:
The reason way is a bit hard to explain here, but this video does a really good job explaining it https://www.youtube.com/watch?v=sVjtp6tGo0g